api/v1alpha1/openapi.yaml

```diff
@@ -2005,6 +2005,36 @@ components:
           description: 'Current state of systemd units on the device.'
           items:
             $ref: '#/components/schemas/DeviceSystemdUnitStatus'
+        applications:
```

Comment:
Do we still need `containers:` when we have this?

Reply:
I agree it should be removed but would require code change if we can agree on the additions in the PR we can remove deprecated fields in a followup which would include code change. I will mark it deprecated for now?

Reply:
systemd also is deprecated and replaced by application. I have marked both as deprecated.